src/components/drag-and-drop/custom-drag-layer/custom-drag-layer.js

```diff
@@ -62,6 +62,10 @@ class CustomDragLayer extends React.Component {
     };
   }
 
+  getKlassName = (props) => {
```

Comment:
I don't think we use `klass` with a `k` anywhere else, might as well stick with `getClassName`?

Reply:
We solve the same problem with the key `class` by using `className` instead (though `klass` is also an alternative in similar situations, `className` is preferred here as it's just a name being defined rather than an object)

Reply:
Correct - will refactor that 👍.